extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1574,6 +1619,40 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     dev().warn(TAG, message, error);
   }
 
+  /**
+   * Generate a 32-byte random string.
+   * Uses the win.crypto when available.
+   * @return {string} The random string
+   * @private
+   */
+  getRandomString_() {
+    // 16 hex characters * 2 bytes per character = 32 bytes
+    const length = 16;
+    let randomValues = Services.cryptoFor(this.win).getSecureRandomBytes(
+      length
+    );
+
+    if (!randomValues) {
+      randomValues = new Array(length);
+      for (let i = 0; i < length; ++i) {
+        randomValues[i] = Math.floor(Math.random() * 255);
+      }
+    }
+
+    // If crypto isn't available, just use Math.random.
```

Comment:
Seems like this comment should be above the if (!randomValues) section, right?

Reply:
done